Essentials/src/com/earth2me/essentials/ISettings.java

```diff
@@ -15,6 +8,12 @@
 import java.util.Set;
 import java.util.function.Predicate;
 import java.util.regex.Pattern;
+import org.bukkit.Material;
```

Comment:
Your formatter moved these imports down. Please match the original file as much as possible. Viewing PRs in GitHub helps.